Refit/MultipartItem.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Refit
+{
+    public abstract class MultipartItem
+    {
+        public MultipartItem()
+        {
+
+        }
+
+        public MultipartItem(string fileName, string contentType)
+        {
+            this.FileName = fileName;
```

Comment:
I think that for the purpose of using the MultiPartItem special type, it's probably better to have FileName required. The parameter name inference is for "low friction" things, but since there's a wrapper type already involved, it may as well be explicit.